src/simplewallet/simplewallet.cpp

```diff
@@ -3691,6 +3805,17 @@ bool simple_wallet::show_transfers(const std::vector<std::string> &args_)
     }
   }
 
+  // subaddr_index
+  std::set<uint32_t> subaddr_indices;
+  if (local_args.size() > 0 && local_args[0].substr(0, 6) == "index=")
+  {
+    if (!parse_subaddress_indices(local_args[0], subaddr_indices))
+      return true;
+    in = pool = true;
+    out = pending = failed = false;
```

Comment:
OK, since each outgoing payment may be associated with multiple subaddresses, I'll let `show_transfers out index=N` print all outgoing payments if some of their inputs come from the specified subaddress.